css/styles.css

```diff
@@ -10,6 +10,7 @@
 html, body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  scroll-behavior: smooth;
```

Comment:
Do we need it?

Reply:
Explained in https://github.com/asyncapi/website/pull/749#discussion_r876207028